code/modules/mob/living/living.dm

```diff
@@ -59,6 +59,15 @@
 		return 0
 	if(!..())
 		return 0
+	if(ishuman(src))
+		var/obj/item/hand_item = src.get_active_hand()
+		if(hand_item && istype(hand_item, /obj/item/weapon/gun) && A != hand_item)
```

Comment:
`hand_item && istype(hand_item` is redundant. Please remove the first conditional, just leaving `istype(hand_item, /obj/item/weapon/gun) && A != hand_item`.